test/models/category_test.rb

```diff
@@ -1,7 +1,49 @@
 require "test_helper"
 
 class CategoryTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  def setup
+    DatabaseCleaner.start
+    site = create(:site)
+    @categories = create_list(:category, 5, site: site)
```

Comment:
> `factory :category` uses sequence for category name.

This is the current behavior and `Category` has no validation for name.
To keep this test stable, I think that to specify category name here is better way.